critiquebrainz/db/oauth_token_test.py

```diff
@@ -69,3 +69,5 @@ def test_get_scopes(self):
             scopes="Test Scopes",
         )
         self.assertIn("Test", db_oauth_token.get_scopes(oauth_token["id"]))
+        db_oauth_token.delete(client_id=self.oauth_client["client_id"], refresh_token="Test Refresh Token")
+        self.assertEqual([], db_oauth_token.get_scopes(oauth_token["id"]))
```

Comment:
I think it would be better if it thrown exception of some kind instead. Because otherwise there's no way to know if the token exists and it has no scope of if there's no token at all.